website/docs/r/virtual_network_gateway_connection.html.markdown

```diff
@@ -245,6 +245,12 @@ The following arguments are supported:
     Site-to-Site or VNet-to-VNet connection is created whereas ExpressRoute
     connections do not need a shared key.
 
+* `connection_protocol` - (Optional) The IKE protocol version to use. Possible
+    values are `IKEv1` and `IKEv2`. Only valid for IPSec connections on virtual
+    network gateways with SKU `VpnGw1`, `VpnGw2`, `VpnGw3`, `VpnGw1AZ`, `VpnGw2AZ`
+    or `VpnGw3AZ`. Changing the protocol will force a new connection to be created. 
```

Comment:
could we split this line:

```
Only valid for IPSec connections on virtual network gateways with SKU `VpnGw1`, `VpnGw2`, `VpnGw3`, `VpnGw1AZ`, `VpnGw2AZ` or `VpnGw3AZ`.
```

into a blue info box below the description? We can do this using:

```
-> Only valid for IPSec connections on Virtual Network Gateways using the SKU `VpnGw1`, `VpnGw2`, `VpnGw3`, `VpnGw1AZ`, `VpnGw2AZ` or `VpnGw3AZ`.
```

in addition could we use the text:

```suggestion
Changing this value forces a new resource to be created.
```

to match the rest of the documentation?